### ISSUE TITLE
Unable to use preplan

### ISSUE THREAD

**Issue by knamsx** [skiselkov/BetterPushbackC#471]
Preplan doesn't receive mouse inputs so I cannot rotate or place anything.

**Comment by twSimulations** [comment]
XP11 or XP12?

**Comment by knamsx** [comment]
XP12 on macOS.

**Comment by bedrich-schindler** [comment]
I have the same issue. Sometimes it works without problem, sometimes I click on pre-plan, view switches to top down view, but I am unable to plan pushback. I can only quit using Escape key. Also help images are not visible.

Using latest X-plane 12 on MacBook Pro M2 MAX with MacOS Ventura 13.3.1

**Comment by hersh81** [comment]
I figured it out for my computer.   I have Tobii eye tracker. If I remove the Tobii plug-in for xplane better pushback works perfectly.  I just can't use tobii eye tracker.

**Comment by bedrich-schindler** [comment]
@skiselkov Is there any chance to check this bug? If you need some assistance or details, just let me know. It stopped working at all. I tried several releases, but none of them make it work.

**Comment by knamsx** [comment]
Yeah, hasn't been great using default pushback on VATSIM, especially with the fact the default push can't turn the Zibo 737 for some reason.

**Comment by SilentDevCoPro** [comment]
Any updates on this? Having the same issue m1 Macbook Pro.  Mouse inputs for planning do not work, so pushback won't work.
